portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,91 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>more about B</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="title">
+      <h1>a page about me</h1>
+      <p>this is my portfolio, a little more about B</p>
     </div>
+
+    <div id="body-content">
+        <div id="about" class="sidebar">
+            <div id="links" class="box">
+                <p2>a crash course in me</p2>
+                <br>
+                <p3>NYU | STEP Intern | art</p3>
+                <br>
+                <a href="https://www.linkedin.com/in/bianca-macias-344a26178">LinkedIn</a>
+                <br>
```

Comment:
```suggestion
                <br/>
```

Reply:
I was wondering about this, as well, and found:
* https://dev.w3.org/html5/html-author/#notes,-tips-and-warnings
* https://google.github.io/styleguide/htmlcssguide.html#Document_Type

Apparently void elements should *not* have the closing /.

Reply:
so leave the break as it is, right?

Reply:
Thanks Chris. Closing the thread.